cloud2sql/parquet.py

```diff
@@ -0,0 +1,175 @@
+from resotoclient.models import Kind, Model, Property, JsObject
+from resotolib.types import Json
+from typing import Dict, Tuple, List, Any
+from resotolib.baseresources import BaseResource
+from resotolib import baseresources
+import pyarrow as pa
+import inspect
+from cloud2sql.util import value_in_path
+import json
+
+# This set will hold the names of all "base" resources
+# Since that are abstract classes, there will be no instances of them - hence we do not need a table for them.
+base_kinds = {
+    clazz.kind for _, clazz in inspect.getmembers(baseresources, inspect.isclass) if issubclass(clazz, BaseResource)
+}
+
+
+carz = [
+    Property("cloud", "string"),
+    Property("account", "string"),
+    Property("region", "string"),
+    Property("zone", "string"),
+]
+carz_access = {name: ["ancestors", name, "reported", "id"] for name in ["cloud", "account", "region", "zone"]}
```

Comment:
Maybe we should move this to a separate model module - it does not belong to SQL or Parquet, but the source data we have.